scrapy/downloadermiddlewares/decompression.py

```diff
@@ -7,13 +7,11 @@
 import zipfile
 import tarfile
 import logging
-from tempfile import mktemp
-
 import six
-
-try:
+from tempfile import mktemp
```

Comment:
a nitpick - it was good to have tempfile import before six, because tempfile is in stdlib. We kind-of consider six to be stdlib as well, because you import stdlib modules using six, but still, in most cases it is below. Sorry for nitpicks! But the PR is about style changes anyways, right? :)

Reply:
Please don't feel sorry for suggesting nitpicks. I am myself kind of suggesting nitpicks too. In fact  I'd be thankful if you keep suggesting them even if they seem trivial to you.